library/ecs_service.py

```diff
@@ -313,29 +333,30 @@ def main():
     module = AnsibleModule(argument_spec=argument_spec, supports_check_mode=True)
 
     if not HAS_BOTO:
-        module.fail_json(msg='boto is required.')
```

Comment:
the stuff in the library folder which is not written by is is not maintained by us.  if we want to we can maintain by pulling in newer versions from the original source